addons/im_livechat/static/tests/livechat_channel_info_list.test.js

```diff
@@ -69,3 +75,84 @@ test("editing livechat note is synced between tabs", async () => {
     await document.querySelector(".o-livechat-ChannelInfoList textarea").blur(); // Trigger the blur event to save the note
     await waitFor(".o-livechat-ChannelInfoList textarea:value('Updated note')", { target: tab1 }); // Note should be synced with bus
 });
+
```

Comment:
Previous test already merged "editing livechat note is synced between tabs" also made the mistake with `target`. Should also change it, because as it is written this cannot be trusted for good working of feature.

Reply:
I don't mind much, but `waitFor()` seems more robust even though it lacks exact match of value and text.
Eventually we'd want to use web test helpers.
But I get the different params is error-prone and can be simpler for now to use `contains()` with familiar signature.